service/orchestrator/certificates.go

```diff
@@ -0,0 +1,172 @@
+package orchestrator
```

Comment:
Unfortunately, it's now hard to see what your changes were because of the changes + extraction, so it might be that some other points come up during this review, which were not your "fault" ;)